code/modules/admin/topic.dm

```diff
@@ -13,566 +13,568 @@
 
 /datum/admins/Topic(href, href_list)
 	..()
-
 	if(usr.client != src.owner || !check_rights(0))
 		message_admins("[usr.key] has attempted to override the admin panel!")
 		log_admin("[key_name(usr)] tried to use the admin panel without authorization.")
 		return
 
 	if(!CheckAdminHref(href, href_list))
 		return
+	switch(href_list["admin_action"])

```

Comment:
I have set it up so it looks for admin_action in the href_list.
admin_action will have the name of whichever admin topic it should be going to saved in it.
Example
`admin_action=stickyban; stickyban=add`